src/explore-education-statistics-admin/src/pages/release/ReleaseMetaGuidancePage.tsx

```diff
@@ -48,11 +52,22 @@ const ReleaseMetaGuidancePage = ({
               caption={model.release.title}
             />
 
-            <h2>Metadata guidance document</h2>
+            <h2>Data guidance</h2>
 
             <ReleaseMetaGuidancePageContent
               published={model.release.published}
               metaGuidance={model.metaGuidance.content}
+              renderDataCatalogueLink={
+                model.release.published ? (
+                  <Link
+                    to={`${PublicAppUrl}/data-catalogue/${model.release.publicationSlug}/${model.release.slug}`}
+                  >
+                    data catalogue
+                  </Link>
+                ) : (
+                  'data catalogue'
```

Comment:
If we have nothing to link to, shouldn't `renderDataCatalogueLink` be set to undefined?

Reply:
I was thinking that as in the admin it's kind of a preview of the page then it made more sense to show the text (and the sentence it's part of) as that gives a more accurate representation of what the public page will look like. Do we have a standard convention for this sort of thing?

Reply:
Not sure, but so far we've generally omitted anything that can't actually be displayed in the admin. I asked Laura if we were planning on putting in an admin data catalogue page and she said no.

It might be worth asking her/Cam what they think about showing this text, even if there is no page that can be displayed!